### PR TITLE
tun module required be zerotier

### PR DESCRIPTION
**Pull request by wltechblog**

### CODE REVIEW

File: package/thingino-kopt/thingino-kopt.mk [hunk 1]
```diff
@@ -316,6 +316,14 @@ define THINGINO_KOPT_LINUX_CONFIG_FIXUPS_NETFILTER
 endef
 endif
 
+# TUN required for zerotier#
+ifeq ($(BR2_PACKAGE_ZEROTIER_ONE),y)
```
Comment (CazYokoyama) [inline package/thingino-kopt/thingino-kopt.mk:320]: Do you expect BR2_PACKAGE_ZEROTIER_ONE=y in local.fragment? Could yo clarify how to enable zerotier?

**Comment by gtxaspec** [comment]
We should consider moving this fixup to the ZeroTier package if it's a dependency of that package. It should still be available as an option in thingino-kopt, but it shouldn't be dependent on ZeroTier. For example, a user might need TUN support without needing ZeroTier

**Comment by themactep** [comment]
yes, fixup for a package should be in the package itself.

**Comment by themactep** [comment]
@wltechblog please check if this change [363a0b90cd97a0f1329b8ff2e44959234419ceeb] works for you.

**Comment by themactep** [comment]
implemented directly in zerotier package.

**Pull request closed without merging**